euler_support/urdf/eef_macro.xacro

```diff
@@ -12,14 +12,14 @@
       <visual>
           <origin xyz="0 0 0" rpy="0 0 0" />
           <geometry>
-                  <mesh filename="package://euler_support/meshes/visual/eef_vacuum.stl" />
+                  <mesh filename="package://euler_support/meshes/visual/motoman_eef.stl" />
```

Comment:
Could we drop the motoman from the eef mesh file names as this part is just a vacuum gripper and not a motoman part?